commcare-cloud/commcare_cloud/commands/inventory_lookup/inventory_lookup.py

```diff
@@ -41,10 +45,32 @@ def run(self, args, ssh_args):
             ssh_args = ['-p', port] + ssh_args
         cmd_parts = [self.command, address] + ssh_args
         cmd = ' '.join(shlex_quote(arg) for arg in cmd_parts)
-        print(cmd)
+        print_command(cmd)
         os.execvp(self.command, cmd_parts)
 
 
 class Mosh(Ssh):
     command = 'mosh'
     help = "Connect to a remote host with mosh"
+
+
+class DjangoManage(CommandBase):
+    command = 'django-manage'
```

Comment:
could you slap on a help message with an example usage?

Reply:
Oh! Yeah, thanks, good catch!

Reply:
https://github.com/dimagi/commcare-cloud/pull/1372